styles/styles.css

```diff
@@ -0,0 +1,337 @@
+@import url('https://fonts.googleapis.com/css?family=Archivo+Black|Merriweather:400,400i');
+
+.flex-container {
+    display: flex;
+    /*Need to decide on columns, rows, or other approach*/
+    /*May also need to figure out if you can do multiple grids in one page and if so whether that is best approach here*/
+}
+
+a:link {
+  text-decoration: none;
+}
+  
+a:visited {
+  text-decoration: none;
+}
+  
+#banner {
+  background-color: #093b60;
+  height: 18em;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+
+}
+
+a.subtitle:visited {
+  color: #093b60;
+}
+
+#letterhead {
```

Comment:
I prefer classes for CSS, and keep the ID's for JS reasons. It's ok if a class is only used once